src/olcs/core.js

```diff
@@ -631,3 +637,17 @@ olcs.core.normalizeView = function(view, angle = 0) {
   view.setRotation(angle);
   view.setResolution(view.constrainResolution(resolution));
 };
+
+/**
+ * Check if the given projection is managed by Cesium (WGS84 or Mercator Spheric)
+ *
+ * @param {ol.proj.Projection} projection Projection to check.
+ * @returns {boolean} If it's managed by Cesium or not.
```

Comment:
The convention is to use `Whether` instead of `if`.
=> Whether the projection is managed by Cesium.